docs/image.rst

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Please don't introduce whitespace changes.

Reply:
sorry, I must've accidentally pressed enter when I was scrolling through the files